include/toBeSorted/area_utils.h

```diff
@@ -1,10 +1,19 @@
 #ifndef AREA_UTILS_H
 #define AREA_UTILS_H
 
+#include "m/m_angle.h"
 #include "m/m_vec.h"
 
-/* 800a9b00 */ bool checkPosInArea(void **AREA, const mVec3_c &pos);
-/* 800a9bd0 */ void *getAreaForIndexInRoom(s32 areaIndex, s32 roomId);
-/* 800a9c50 */ bool *checkPosInArea(s32 areaIndex, s32 roomId, const mVec3_c &pos, void **AREA);
+struct AREA {
```

Comment:
Is this Area the one seen in bzs?
We should probably have proper BZS entries somewhere

Reply:
I think so, yes. Let's do that in a separate PR though because I haven't really looked at BZS yet

Reply:
Sounds good